src/v/cluster/rm_stm.cc

```diff
@@ -65,7 +65,8 @@ static model::record_batch make_fence_batch(model::producer_identity pid) {
 static model::record_batch make_fence_batch(
   model::producer_identity pid,
   model::tx_seq tx_seq,
-  std::chrono::milliseconds transaction_timeout_ms) {
+  std::chrono::milliseconds transaction_timeout_ms,
```

Comment:
fixed

Reply:
it was convenient to name it `make_fence_batch_last` to avoid collision during refactoring but now it's better to switch back to `make_fence_batch`. `make_fence_batch_v*` are temporary methods existing only to provide backward compatibility during the upgrade. eventually we'll remove all `make_fence_batch_v*` methods and `make_fence_batch` makes more sense than `make_fence_batch_last`. We'll drop `make_fence_batch_v0` in 23.2 and `make_fence_batch_v1` most likely in 23.3